pkg/modprobe/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 as mirror
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    musl \
+    && true
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+RUN mkdir /out/bin && mkdir /out/sbin && cp /bin/busybox /out/bin/busybox && cp -P /sbin/modprobe /out/sbin/modprobe
```

Comment:
@rn I am now correctly copying the `/sbin/modprobe` symlink with `cp -P` in the first stage in lieu of creating the symlink in the second stage. The `CMD ["/bin/busybox", ...]` is no longer needed.

Reply:
another super minor nit but I prefer a command per line so using a `&& \` and then move to the next line is preferred